### PR TITLE
Raise WP version to latest stable in unit tests matrix

### PR DESCRIPTION
**Pull request by imath**
Upgrade WP Tested version to 6.5

Trac ticket: https://buddypress.trac.wordpress.org/ticket/9132

---
**This Pull Request is for code review only. Please keep all other discussion in the BuddyPress Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the WordPress Core Handbook for more details.**

### CODE REVIEW

File: .github/workflows/unit-tests.yml [hunk 1]
```diff
@@ -20,9 +20,9 @@ jobs:
         wp_version: ['master']
         include:
           - php: '8.0'
-            wp_version: '6.4'
+            wp_version: '6.5'
```
Comment (renatonascalves) [inline .github/workflows/unit-tests.yml:23]:
```suggestion
            wp_version: 'latest'
```

File: .github/workflows/unit-tests.yml [hunk 2]
```diff
@@ -20,9 +20,9 @@ jobs:
...
           - php: '7.4'
-            wp_version: '6.4'
+            wp_version: '6.5'
```
Comment (renatonascalves) [inline .github/workflows/unit-tests.yml:25]:
```suggestion
            wp_version: 'latest'
```

**Review by renatonascalves — APPROVED** [review]
This will avoid another bump when 6.6 is released.

**Comment by imath** [comment]
Hi @renatonascalves 

Thanks for the tip! Actually it wasn't `'latest'` but `null` to use inside the wp-env override JSON file.
See: https://developer.wordpress.org/block-editor/reference-guides/packages/packages-env/#wp-env-json

**Comment by renatonascalves** [comment]
TIL. Nice!

**Pull request closed without merging**